background-remover/background-remover.py

```diff
@@ -0,0 +1,20 @@
+import os
+import rembg
+
+def process (input_path: str, output_path: str, session = rembg.new_session()):
```

Comment:
if using type annotations for arguments add one for return type

```python
def process(input_path: str, output_path: str, session = rembg.new_session()) -> bool:
```
if I'm not mistaken

Reply:
oh yeah, will add that